src/Behavior/MillisecondTimestamp.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace ipl\Orm\Behavior;
+
+use DateTime;
+use DateTimeZone;
+use Exception;
+use ipl\Orm\Contract\PropertyBehavior;
+use ipl\Orm\Exception\ValueConversionException;
+
+class MillisecondTimestamp extends PropertyBehavior
+{
+    public function fromDb($value, $key, $context)
+    {
+        if ($value === null) {
+            return null;
+        }
+
+        $datetime = DateTime::createFromFormat('U.u', sprintf('%F', $value / 1000.0));
+        $datetime->setTimezone(new DateTimeZone(date_default_timezone_get()));
+
+        return $datetime;
+    }
+
+    public function toDb($value, $key, $context)
+    {
+        if (is_numeric($value)) {
+            return sprintf('%F', $value * 1000.0);
```

Comment:
A minor nitpick maybe, but I just noticed that this always transmits strings with decimals. `fromDB` however expects integers.

Is there a reason I forgot why you're using sprintf here? Didn't the result of the multiplication work?